### PR TITLE
reduce pr summary comment to 3 wf/addon and 5 unit tests

### PR DESCRIPTION
**Pull request by smuzaffar**

### CODE REVIEW

File: pr_testing/setup-pr-test-env.sh [hunk 1]
```diff
@@ -26,7 +26,7 @@ if [ ! -f $WORKSPACE/prs_commits ] ; then
 fi
 cp -f $CMS_BOT_DIR/das-utils/das_client $CMS_BOT_DIR/das-utils/das_client.py
 if ! which scram >/dev/null 2>&1 ; then source /cvmfs/cms.cern.ch/cmsset_default.sh ; fi
-export CMS_PATH=/cvmfs/cms-ib.cern.ch
+export CMS_PATH=/cvmfs/cms.cern.ch
```
Comment (smuzaffar) [inline pr_testing/setup-pr-test-env.sh:29]: this is to force failed the tests to see the summary comment here

**Comment by smuzaffar** [comment]
test parameters:
 - full_cmssw = true

**Comment by smuzaffar** [comment]
please test

**Comment by smuzaffar** [comment]
please test

**Comment by smuzaffar** [comment]
please test

**Comment by smuzaffar** [comment]
please test

**Comment by smuzaffar** [comment]
please test

**Comment by smuzaffar** [comment]
please test

**Comment by smuzaffar** [comment]
please test

**Comment by smuzaffar** [comment]
please test

**Comment by smuzaffar** [comment]
please test

**Comment by smuzaffar** [comment]
please test

**Comment by smuzaffar** [comment]
please test

**Comment by smuzaffar** [comment]
please test

**Comment by smuzaffar** [comment]
please test

**Comment by smuzaffar** [comment]
@slava77 , how about this https://github.com/cms-sw/cms-bot/pull/1473#issuecomment-765586774 format? Now bot shows expanded errors for first 3 workflows/addons and rest are collapsed.

**Comment by slava77** [comment]
> @slava77 , how about this [#1473 (comment)](https://github.com/cms-sw/cms-bot/pull/1473#issuecomment-765586774) format? Now bot shows expanded errors for first 3 workflows/addons and rest are collapsed.

it looks nice.

Some of the explicitly quoted items could be compacted, perhaps. e.g. instead of 
* 4.53
```
4.53_RunPhoton2012B+RunPhoton2012B+HLTD+RECODR1reHLT+HARVESTDR1reHLT/step1_dasquery.log
```

* 4.53 ```4.53_RunPhoton2012B+RunPhoton2012B+HLTD+RECODR1reHLT+HARVESTDR1reHLT/step1_dasquery.log```

Wehere the latter is made with escape backquotes on the same line as the item listed (without extra new lines)
Screen Shot 2021-01-22 at 10 50 12 AM


Thank you.

**Comment by smuzaffar** [comment]
thanks, I will update.

**Comment by smuzaffar** [comment]
please test

**Comment by slava77** [comment]
I don't have any comments to the last result at this point.

**Pull request merged**